packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -107,6 +108,8 @@ export const App = ({
               <ResetPassword path="/reset_password/*" />
               <ConfirmResetPassword path="/confirm_reset_password/*" />
 
+              <WebChannelExamples path="/web_channel_examples/*" />
```

Comment:
Is there a way to block this from loading in production?

Reply:
Good question. Should be doable with config settings.

Reply:
@vbudhram I've added a config that disables the route by default and the component returns nothing if the env isn't development.